packages/augur-ui/Dockerfile

```diff
@@ -1,79 +1,22 @@
-ARG NODE_VERSION=10.15.0
-FROM node:${NODE_VERSION}-alpine as builder
-
-ENV PATH /root/.yarn/bin:$PATH
-ARG ethereum_network=rinkeby
-ENV ETHEREUM_NETWORK=$ethereum_network
+FROM augurproject/augur-build:latest as augur-build
 ARG build_environment=dev
 ENV BUILD_ENVIRONMENT=$build_environment
-RUN apk --no-cache add \
-    bash \
-    binutils \
-    eudev-dev \
-    g++ \
-    git \
-    libusb-dev \
-    linux-headers \
-    make \
-    python
+ARG ethereum_network=rinkeby
+ENV ETHEREUM_NETWORK=$ethereum_network
 
 
-# begin create caching layer
-COPY package.json /augur/package.json
-ADD https://nodejs.org/download/release/v${NODE_VERSION}/node-v${NODE_VERSION}-headers.tar.gz /augur/node-v${NODE_VERSION}-headers.tar.gz
 WORKDIR /augur
-RUN git init \
-  && yarn add require-from-string \
-  && yarn \
-  && rm -rf .git \
-  && rm package.json \
-  && rm yarn.lock
-# end create caching layer
-
-COPY . /augur
-COPY support/local-run.sh /augur/local-run.sh
-
-# workaround a bug when running inside an alpine docker image
-RUN rm -f /augur/yarn.lock
 
 RUN set -ex; \
     if [ "$BUILD_ENVIRONMENT" = "dev" ]; then \
-        ETHEREUM_NETWORK=$ethereum_network yarn build --dev --augur-hosted --disableMainnet; \
+        ETHEREUM_NETWORK=$ethereum_network yarn workspace @augurproject/ui build --dev --augur-hosted --disableMainnet; \
```

Comment:
Is `ETHEREUM_NETWORK` already defined? If so, we will need to install git above.